src/main/kotlin/highcharts/Highcharts.kt

```diff
@@ -216,6 +216,6 @@ external class Highcharts {
     fun chart(renderTo: String, opts: Options): Chart
 }
 
-@JsModule("HighCharts")
+@JsModule("highcharts")
```

Comment:
This is really surprising to me, as obviously this was working before, and I remember having to do work to get this going.  I'm pretty sure `Highcharts` was due to the type I exposed [here](https://github.com/jitsi/jvb-dashboard/blob/master/src/main/kotlin/highcharts/Highcharts.kt). Nothing else has been upgraded here or anything, right?  What was the error you were seeing, @gpolitis ?